dubbo-samples-rest/README.md

```diff
@@ -28,12 +28,12 @@ We can use two steps to export Rest Service in Dubbo:
 Step 1: Add dependency to the pom file in our project:
 ```
         <dependency>
-            <groupId>org.apache</groupId>
+            <groupId>com.alibaba</groupId>
```

Comment:
I think make the  groupId as a placeplaceolder is better, its value is as you explained below.

Reply:
I have made changes according to your suggestion.